src/Evolution/DgSubcell/Tags/DidRollback.hpp

```diff
@@ -0,0 +1,16 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+
+namespace evolution::dg::subcell::Tags {
+/// Set by the DG scheme when the predicted step failed and a rollback is
+/// performed. The subcell solver checks the tag, and uses the DG boundary data
+/// if a rollback occurred in order to maintain conservation. The subcell solver
+/// then sets `DidRollback` to `false`.
```

Comment:
- I suggest "Set" -> "Set to `true`" to give a little more info
- I optionally also suggest adding a new sentence like "Tag indicating whether we are retrying a step after a rollback of a failed DG step" as a brief, then the current paragraph giving all the juicy details.